software/verification/verify_cdl_sequence.py

```diff
@@ -206,7 +240,8 @@ def execute_tests(self):
                                                                            actual_output=actual_output,
                                                                            output_config=output_variable_configurations[op],
                                                                            variable=device_point_name,
-                                                                           unit=cdl_point_unit)
+                                                                           unit=cdl_point_unit,
+                                                                           output_folder=model_dir)
                     print("\n")
             if validation:
                 print("SUCCESSFULLY VERIFIED SEQUENCE! ")
```

Comment:
Better would be something like "Sequence conforms to specification." because a sequence can be verified successful, and the result of the verification is that the sequence fails to meet the specification. E.g., it is not the "verification" that is successful as a verification can return failed, succeeded or not tested, but rather it is whether the sequence passed the verification tests.